web/posts/2018-07-29-how-i-use-stuff-day-one.markdown

```diff
@@ -0,0 +1,184 @@
+have to do.
+
+[microblog-ifttt]: https://ifttt.com/applets/52590350d-create-new-journal-entries-from-microblog-posts
+
+### Reading (38 Entries)
+This is also populated via IFTTT, and is intended to be a sort of Commonplace
+Book, where all interesting quotes from things I read are kept. Unfortunately,
+it's incomplete right now because I can only populate it via Instapaper. My
+Kindle notes/highlights are siloed away inside Amazon's data centers, and they
+don't seem to be particularly interested in letting them out. It's a real
+shame, I'd love to be able to export those easily (and automatically).
+
+As it stands, any time I make a highlight in Instapaper [an IFTTT applet
+fires][instapaper-ifttt] and an entry is added here.
+
+[instapaper-ifttt]: https://ifttt.com/applets/68248714d-if-new-highlight-then-create-journal-entry
+
+## On Journals and Tags
+One common point of confusion/contention with Day One seems to be when to use
+journals and when to use tags. The approach I’ve taken is to create journals
+when I’m posting about specific thing often enough that it begins to clutter
+my main Journal. So for example, I wrote my productivity entries inside
+Journal until I decided to do it every day. At that point, the entries started
+cluttering that journal and so I moved them all out into their own
+Productivity journal. This gave these entries a place to live while also
+removing the noise from my main Journal.
+
+Tags, on the other hand, I’m fairly loose with. I create them almost at will.
+I have some common ones that I add to larger topics (for example,
+`travel/2018/europe` or `work/square`), but others are added arbitrarily. I
+don’t put too much time into thinking about them, and don’t have a great
+strategy for how to manage them.
+
+## Wishlist
+You might have noticed that I'm using a _number_ of tools to overcome some
+shortcomings of Day One itself. I'd love to reduce the number of external
+tools I use to make my practice work, but I'd first need some improvements to
+Day One:
+
+* Template Support. Ideally, templates with multiple prompts. This would
+  improve so much about my practice. One of my biggest pain-points right now
+  is that since Workflow is on iOS, it means I have to switch to my iPhone in
+  order to do those daily/weekly productivity entries. This is especially
+  annoying for the weekly entries since those can get long, and I'd much
+  rather write those on a full keyboard instead of in an alert on my iPhone.
+* External API. This can be (should be?) write only, and could (should?) work
+  only with unencrypted journals, but direct API access would let me feel
+  _much_ better about things like my microblog entries. Being able to control
+  the content without needing to go through IFTT would mean I could ensure
+  that my entries are formatted consistently. It would also fix the fact that
+  IFTTT "helpfully" strips HTML from any passed text.
```

Comment:
I imagine you've poked around with this? http://help.dayoneapp.com/tips-and-tutorials/command-line-interface-cli